rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -72,19 +61,93 @@ def _compute_features(
         self, batch_examples: List[Message], attribute: Text = TEXT_ATTRIBUTE
     ) -> np.ndarray:
 
+        sentence_encodings = self._compute_sentence_encodings(batch_examples, attribute)
+
+        return self._compute_sequence_encodings(
+            batch_examples, sentence_encodings, attribute
+        )
+
+    def _compute_sentence_encodings(
+        self, batch_examples: List[Message], attribute: Text = TEXT_ATTRIBUTE
+    ) -> np.ndarray:
         # Get text for attribute of each example
         batch_attribute_text = [ex.get(attribute) for ex in batch_examples]
+        sentence_encodings = self._sentence_encoding_of_text(batch_attribute_text)
 
-        batch_features = self._run_model_on_text(batch_attribute_text)
+        # convert them to a sequence of 1
+        return np.reshape(sentence_encodings, (len(batch_examples), 1, -1))
 
-        return batch_features
+    def _compute_sequence_encodings(
```

Comment:
Optional suggestion: 
Add the pre-computed sentence encodings to the list of sequence encodings outside of this function since the name of the function doesn't suggest to do that.